Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -86,13 +86,25 @@ public function registerNodeChange(NodeInterface $node)
     }
 
     /**
-     * Fetches possible usages of the asset and registers nodes that use the asset as changed.
+     * Deprecated. Please use ContentCacheFlush::registerAssetChange
      *
+     * @deprecated
      * @param AssetInterface $asset
      * @return void
      */
     public function registerAssetResourceChange(AssetInterface $asset)
     {
+        $this->registerAssetChange($asset);
+    }
+
+    /**
+     * Fetches possible usages of the asset and registers nodes that use the asset as changed.
+     *
+     * @param AssetInterface $asset
+     * @return void
+     */
+    public function registerAssetChange(AssetInterface $asset)
+    {
         if (!$asset->isInUse()) {
```

Comment:
Here is an issue with the interface. The `AssetInterface` does not contain the `isInUse `method. Any intention to change the interface here? Or should we silmly change the Parameter to `Asset`?